low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -23,15 +23,18 @@ import sys
 import shutil
 import json
 import subprocess
+import glob
 
 from framework.base.sspl_constants import (PRODUCT_FAMILY, SUPPORT_REQUESTOR_NAME,
     SUPPORT_EMAIL_ID, SUPPORT_CONTACT_NUMBER, ENCL_TRIGGER_LOG_MAX_RETRY,
-    ENCL_DOWNLOAD_LOG_MAX_RETRY, ENCL_DOWNLOAD_LOG_WAIT_BEFORE_RETRY, CONSUL_DUMP,
-    DATA_PATH)
+    ENCL_DOWNLOAD_LOG_MAX_RETRY, ENCL_DOWNLOAD_LOG_WAIT_BEFORE_RETRY, DATA_PATH)
 from framework.utils.conf_utils import Conf, SSPL_CONF
 
-onGoingRequestPath = f"{DATA_PATH}sspl-sb-req.txt"
-sspl_bundle_tmp = f"{DATA_PATH}sspl_bundle_{str(int(time.time()))}"
+SB_DATA_PATH = DATA_PATH + 'support_bundle'
+if not os.path.exists(SB_DATA_PATH):
+    os.makedirs(SB_DATA_PATH)
```

Comment:
You can simplify these two lines to one.
`os.makedirs(SB_DATA_PATH, exist_ok=True)`
Apply it wherever applicable.

Reply:
Done